roulier/carriers/laposte_fr/api.py

```diff
@@ -314,7 +314,7 @@ def _customs(self):
                     },
                     "email": {
                         "type": "string",
-                        "regex": "^\\w*@\\w*\.\\w*$",
+                        "regex": r"^\w*@\w*.\w*$",
```

Comment:
Let's use a better one. 
```suggestion
                        # according to https://emailregex.com/
                        "regex": r"(^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$)",
```